### PR TITLE
Cloud formation correction

### PR DESCRIPTION
**Pull request by markgov**
Correction to the AWS code for the cloud formation as there was a failure on apply. This Pr corrects the error of duplicate path sections in the cloud formation code and add back some missing stack set IAM rules

### CODE REVIEW

**Comment by davidkelliott** [comment]
Could you highlight what has changed please as it's hard to tell what has changed vs what are formatter changes, thanks

File: organisation-security/terraform/cloudformation/OracleDbLTS-Orchestrate.yaml [hunk 1]
```diff
@@ -123,42 +119,66 @@ Resources:
...
-                  - 'iam:PassRole'
-                Resource: 
-                  - !Sub 'arn:${AWS::Partition}:iam::${AWS::AccountId}:role/OracleDbLTS-SystemsManagerAutomationAdministrationRole'
-                  - !Sub 'arn:${AWS::Partition}:iam::*:role/OracleDbLTS-SystemsManagerAutomationExecutionRole'
-  
+                  - "iam:PassRole"
+                Resource:
+                  - !Sub "arn:${AWS::Partition}:iam::${AWS::AccountId}:role/OracleDbLTS-SystemsManagerAutomationAdministrationRole"
+                  - !Sub "arn:${AWS::Partition}:iam::*:role/OracleDbLTS-SystemsManagerAutomationExecutionRole"
+  StackSetAdministrationRole:
```
Comment (markgov) [inline organisation-security/terraform/cloudformation/OracleDbLTS-Orchestrate.yaml:157]: Added This back into the template as it was not present in the yaml file that i got from aws

File: organisation-security/terraform/cloudformation/OracleDbLTS-Orchestrate.yaml [hunk 2]
```diff
@@ -207,23 +225,35 @@ Resources:
           - Effect: Allow
             Principal:
               Service: [lambda.amazonaws.com]
-            Action: ['sts:AssumeRole']
+            Action: ["sts:AssumeRole"]
       ManagedPolicyArns:
-        - !Sub 'arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole'
-      Path: /
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"
```
Comment (markgov) [inline organisation-security/terraform/cloudformation/OracleDbLTS-Orchestrate.yaml:230]: There was a duplicate Path: / on the line bellow which was causing the cloud formation to fail

File: organisation-security/terraform/cloudformation/OracleDbLTS-Orchestrate.yaml (hunk 1 above)
Comment (markgov) [reply organisation-security/terraform/cloudformation/OracleDbLTS-Orchestrate.yaml:157]: that lines 157 to 180

File: organisation-security/terraform/cloudformation/OracleDbLTS-Orchestrate.yaml [hunk 3]
```diff
@@ -246,46 +276,112 @@
...
+                - !Sub "arn:${AWS::Partition}:iam::${AWS::AccountId}:root"
+            Action:
+              - sts:AssumeRole
+      Policies:
+        - PolicyName: ExecutionPolicy
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Sid: Sid0
+                Effect: Allow
```
[the remark this answers is not in the document]
Comment (ewastempel) [reply organisation-security/terraform/cloudformation/OracleDbLTS-Orchestrate.yaml:311]: I cannot locate any secret in here.

File: organisation-security/terraform/cloudformation/OracleDbLTS-Orchestrate.yaml (hunk 1 above)
Comment (ewastempel) [reply organisation-security/terraform/cloudformation/OracleDbLTS-Orchestrate.yaml:157]: looks good

File: organisation-security/terraform/cloudformation/OracleDbLTS-Orchestrate.yaml (hunk 2 above)
Comment (ewastempel) [reply organisation-security/terraform/cloudformation/OracleDbLTS-Orchestrate.yaml:230]: makes sense

**Review by ewastempel — APPROVED** [review]
LGTM

**Pull request merged**